src/components/KRadio/KRadio.vue

```diff
@@ -1,54 +1,79 @@
 <template>
   <div
-    :checked="isSelected"
     class="k-radio"
     :class="[
-      isTypeDefault ? 'k-radio-default' : `k-radio-${type}`,
       $attrs.class ? $attrs.class : '',
-      { 'disabled': isDisabled }
+      { 'disabled': isDisabled, 'radio-card': isCard || type === 'card', 'has-error': hasError, 'checked': isChecked, 'has-description': showDescription }
     ]"
   >
     <input
       :id="inputId"
-      :checked="isSelected"
+      :checked="isChecked"
       v-bind="modifiedAttrs"
-      class="k-input"
+      class="radio-input"
       :disabled="isDisabled"
       type="radio"
       @click="handleClick"
     >
 
-    <KLabel
-      v-if="isTypeDefault && hasLabel"
-      v-bind="labelAttributes"
-      class="k-radio-label"
-      :class="{ 'has-description': hasDescription }"
-      :for="inputId"
+    <div
+      v-if="!isCard && (label || $slots.default)"
+      class="radio-label-wrapper"
+      :class="{ 'has-description': showDescription }"
     >
-      <slot>{{ label }}</slot>
+      <KLabel
+        v-bind="labelAttributes"
+        class="radio-label"
+        :for="inputId"
+      >
+        <slot>{{ label }}</slot>
+
+        <template
+          v-if="hasTooltip"
+          #tooltip
+        >
+          <slot name="tooltip" />
+        </template>
+      </KLabel>
 
       <div
-        v-if="hasDescription"
-        class="k-radio-description"
+        v-if="showDescription"
+        class="radio-description"
       >
         <slot name="description">
-          {{ description }}
+          <p>{{ description }}</p>
         </slot>
       </div>
-
-      <template
-        v-if="hasTooltip"
-        #tooltip
-      >
-        <slot name="tooltip" />
-      </template>
-    </KLabel>
+    </div>
 
     <label
-      v-else-if="$slots.default"
+      v-else-if="label || $slots.default"
+      class="radio-card-wrapper radio-label-wrapper"
+      :class="{ 'has-label': label, 'has-description': showCardDescription }"
       :for="inputId"
+      :tabindex="isDisabled ? -1 : 0"
+      @keyup.space="handleClick"
```

Comment:
issue: on some pages, pressing `space` here will also scroll the page (e.g. on the docs site).

Do we need to add a `.prevent` modifier to the event?